docs/modules/session-explorer/partials/launch-session-explorer.adoc

```diff
@@ -2,12 +2,12 @@
 
 In Kobiton, select *Sessions*.
 
-image:session-explorer:select-sessions-closeup.png[width=1000,alt="Select Sessions tab"]
+image:session-explorer:select-sessions-closeup.png[width=150,alt="Select Sessions tab"]
 
 xref:session-explorer:search-for-a-session.adoc[Find a session], then select the session in the search result.
 
 image:session-explorer:search-session-id-closeup.png[width=1000,alt="Search for a session ID"]
 
-At the session details page, select the *Explorer* tab.
+Once you select your session, the *Session Explorer* page will appear.
```

Comment:
change "the Session Explorer page" to "Session Explorer" (can you search and see if other places? I may not have caught all)